micro-rdk/build.rs

```diff
@@ -1,6 +1,7 @@
 use regex::Regex;
 
 fn main() {
+    println!("cargo::rustc-check-cfg=cfg(esp32)");
```

Comment:
Like above, clippy would complain that this was an unknown config in esp32/board.rs.